### ISSUE TITLE
PW Sprint 73 plan

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#11705]
### 1. Harden homepage for prod

**Page itself:**
- [x] #11389 (5)
- [ ] #11704 

**Modal:**
  - [ ] #11682 ()
  - [ ] #11653 (3)
  - [ ] #11535 () 🥅 
  - [ ] NEW TICKET NEEDED: New cookie behavior for announcements framework
 

**Drupal**
  - [ ] (stretch pending Blake input/CMS Collab cycle) #11086 ()
  - [ ] NEW TICKET NEEDED: cms collab cycle intake

**Post-MVP, not launch blocking (and not in Sprint)**
  - #11569 
  - #11568 (2)

**What else?**
  - Launch tickets:
    - TBD QA feedback, any new tickets
    - #11652   - Michelle content entry - #11370
  - #11286 (Jill to check, presume no-op)
  - #11586 ()

### 2. PDF Certificate Expiration Alert
- [X] #11021 (3) 🥅  - done, ready for staging review
- [X] #11022 (5)  - done, ready for staging review
- [ ] (stretch; depends on Forms reaching 100% updated) #11616
 
### 3. Income Limits web app
- [ ] #11755 () 
- [ ] #11643 () 🥅 

### 4. Chatbot 
- No action yet identified

### 5. Benefit Content Model Hardening
- Dependency on Danielle prioritizing

### 6. Cerner API deprecation support
- [X] #9179 (8) - no Plat approval yet, may slip 12/14 date
- [x] #11658 

###  Other 508
- #11205 - tabled for deeper holistic approach later in Q4 / early Q1

---

### Redirects
- [ ] #11662 - TBD how to, need Dave's input
- [ ] #10972 (2)(pending unblocking by stakeholders)
- [x] #11642
- [ ] #11487 
- [ ] #11738
- [ ] #11730
- [ ] #11745 
- [ ] #11786 

### Tech debt

- [X] #10342 do-over
- [ ] (audit + fix due Dec 7) #11358 - confirming timing [in Slack](https://dsva.slack.com/archives/C03R5SBELQM/p1670013407405859?thread_ts=1666879267.774979&cid=C03R5SBELQM)

### Content support
- [X] #11420  (2) - have proposed moving to GovCIO board, since they're blocking merge

### Form audit tools
- [X] #11164 (3)
- [ ] #11532 (2)
- [ ] #11500 (3)

### Drupalize footer
- [ ] #11482

### Injected header post-mortem
- Actions waiting on conclusion of post-mortem
